server/gameconnection.py

```diff
@@ -246,20 +257,33 @@ def send_message(self, message):
         army = int(army)
         result = str(result)
         try:
-            if not any([result.startswith(possible_result)
-                        for possible_result in ['score', 'defeat', 'victory', 'draw']]):
+            if not any(
+                [
+                    result.startswith(possible_result)
+                    for possible_result in [
+                        "score",
+                        "defeat",
+                        "victory",
+                        "draw",
+                    ]
+                ]
+            ):
```

Comment:
well, the code, that was before was valid and nice, rewritting loop like that makes me unhappy. in fact, you want code to be compact, because of reading.